backend/tracim_backend/lib/core/content.py

```diff
@@ -1907,17 +1908,33 @@ def update_file_data(
         item.revision_type = ActionDescription.REVISION
         return item
 
-    def check_upload_size(self, content_length: int):
+    def check_upload_size(self, content_length: int, workspace: Workspace) -> None:
+        self._check_size_length_limitation(content_length)
+        self._check_workspace_size_limitation(content_length, workspace)
+
+    def _check_size_length_limitation(self, content_length: int) -> None:
         # INFO - G.M - 2019-08-23 - 0 mean no size limit
         if self._config.LIMITATION__CONTENT_LENGTH_FILE_SIZE == 0:
-            return
+            pass
         elif content_length > self._config.LIMITATION__CONTENT_LENGTH_FILE_SIZE:
             raise FileSizeOverMaxLimitation(
                 'File cannot be added because his size "{}" is higher than max allowed size : "{}"'.format(
                     content_length, self._config.LIMITATION__CONTENT_LENGTH_FILE_SIZE
                 )
             )
 
+    def _check_workspace_size_limitation(self, content_length: int, workspace: Workspace) -> None:
+        workspace_size = workspace.get_size()
+        # INFO - G.M - 2019-08-23 - 0 mean no size limit
+        if self._config.LIMITATION__WORKSPACE_SIZE == 0:
+            pass
```

Comment:
I prefer `return` here, to avoid execution of any future code added in function.

Reply:
done.